ddl/serial_test.go

```diff
@@ -338,6 +338,25 @@ func (s *testSerialSuite) TestCreateTableWithLike(c *C) {
 	tk.MustExec("create table ctwl_db1.pt1 like ctwl_db.pt1;")
 	tk.MustQuery("select * from ctwl_db1.pt1").Check(testkit.Rows())
 
+	// for pre_split_table
```

Comment:
The test doesn't test this PR, the master branch can pass this test too.

Reply:
You may need test like `TestShowTableRegion`.

Reply:
Add test show table region